packages/react-charting/src/components/DonutChart/__snapshots__/DonutChart.test.tsx.snap

```diff
@@ -63,6 +63,7 @@ exports[`DonutChart - mouse events Should render callout correctly on mouseover
                   }
               d="M0.585,-54.997A55,55,0,0,1,51.383,-19.615L-18.955,6.382Z"
               data-is-focusable={true}
+              fill="#E5E5E5"
```

Comment:
why is this fill color showing for non gradient enabled test

Reply:
The line `fill={this.props.enableGradient ? 'transparent' : this.props.color}` in Arc.tsx is causing this.
Previously the fill was set by CSS classname only, but now its being done by the fill property in the JSX. 
Adding gradient enabled tests shortly

Reply:
Can we do it using css with gradients as well

Reply:
To be taken in separate PR